src/test/java/com/puppycrawl/tools/checkstyle/ant/CheckstyleAntTaskTest.java

```diff
@@ -195,9 +194,9 @@ public final void testPathsDirectoryWithNestedFile() throws IOException {
                 TestRootModuleChecker.isProcessed());
         final List<File> filesToCheck = TestRootModuleChecker.getFilesToCheck();
         assertThat("There more files to check then expected",
-                filesToCheck.size(), is(1));
+                filesToCheck.size(), is(9));
         assertThat("The path of file differs from expected",
-                filesToCheck.get(0).getAbsolutePath(), is(getPath(FLAWLESS_INPUT)));
+                filesToCheck.get(5).getAbsolutePath(), is(getPath(FLAWLESS_INPUT)));
```

Comment:
@subkrish Thats just the profile. It is similar to code coverage in that you will get a build failure if you don't have the right percentage of coverage, but pitest isn't set to 100% yet.